x-pack/plugins/monitoring/server/lib/alerts/fetch_index_shard_size.test.ts

```diff
@@ -146,6 +185,29 @@ describe('fetchIndexShardSize', () => {
       },
     ]);
   });
+  it('higher alert threshold', async () => {
+    esClient.search.mockResponse(
+      // @ts-expect-error not full response interface
```

Comment:
Let's see if we can remove this ts-expect-error? If we change how the `esRes` [is assigned ](https://github.com/elastic/kibana/pull/143751/files#diff-b2a51711a58afd05edb8b77eddb360603121e5cfac1028c4ca41dc63df90a05dL37), we might be able to do that:

```
const esRes: estypes.SearchResponse = {
    took: 1,
    timed_out: false,
    _shards: {
      total: 0,
      successful: 0,
      failed: 0,
      skipped: 0,
    },
    hits: {
      total: 0,
      max_score: 0,
      hits: [],
    },
    aggregations: {...}
```